lib/python/bind_units.cpp

```diff
@@ -87,6 +89,33 @@ units::Unit from_dict(const py::dict &dict) {
       dict["multiplier"].cast<double>()));
 }
 
+void format_base(std::ostringstream &oss, const std::string_view base,
+                 const int power) {
+  if (power != 0)
+    oss << "*" << base << "**" << power;
+}
+
+std::string repr(const units::Unit &unit) {
+  assert_simple_unit(unit);
+
+  std::ostringstream oss;
+  oss << unit.underlying().multiplier();
+
+  const auto &&base_units = unit.underlying().base_units();
+  format_base(oss, "m", base_units.meter());
+  format_base(oss, "kg", base_units.kg());
+  format_base(oss, "s", base_units.second());
+  format_base(oss, "A", base_units.ampere());
+  format_base(oss, "K", base_units.kelvin());
+  format_base(oss, "mol", base_units.mole());
+  format_base(oss, "cd", base_units.candela());
+  format_base(oss, "$", base_units.currency());
+  format_base(oss, "rad", base_units.radian());
```

Comment:
Forgot count? Maybe this design is a bit risky?

Reply:
Yes, I forgot it.
What is risky about this? The manual listing? I'd be happy to see a *simple* alternative in C++...

Reply:
Yes, the manual listing... maybe we can at least try to avoid doing it multiple times (see above in `to_dict`)?